app/controllers/cart_items_controller.rb

```diff
@@ -31,7 +31,7 @@ def show
 
   def destroy
   item = Item.find(params[:id].to_i)
-   if @cart.contents.delete_if { |item_id,quantity| item_id == params[:id] }
+  if @cart.contents.delete_if { |item_id,quantity| item_id == params[:id] }
```

Comment:
Unused block argument - `quantity`. If it's necessary, use `_` or `_quantity` as an argument name to indicate that it won't be used.<br>Space missing after comma.